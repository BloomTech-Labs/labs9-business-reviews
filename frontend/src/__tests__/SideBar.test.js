import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import SideBarComponent from '../components/SideBar';

describe('PopularBusinessesComponent />', () => {
  it('renders', () => {
    shallow(<SideBarComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<SideBarComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
