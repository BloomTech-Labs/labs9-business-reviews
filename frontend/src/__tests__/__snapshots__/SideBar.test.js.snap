// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopularBusinessesComponent /> matches snapshot 1`] = `
<styled.div>
  <a
    href="/search"
  >
    Search
  </a>
  <a
    href="/myreviews"
  >
    My Reviews
  </a>
  <Link
    replace={false}
    to="/billing"
  >
    Billing
  </Link>
  <a
    href="settings"
  >
    Settings
  </a>
</styled.div>
`;
